src/replica/mutation_log.h

```diff
@@ -76,67 +76,73 @@ class mutation_log : public ref_counter
     typedef std::function<void(dsn::error_code err)> io_failure_callback;
 
 public:
-    // append a log mutation
-    // return value: nullptr for error
-    // thread safe
+    // Append a log mutation.
+    // Return value: nullptr for error.
+    //
+    // Thread safe.
     virtual ::dsn::task_ptr append(mutation_ptr &mu,
                                    dsn::task_code callback_code,
                                    dsn::task_tracker *tracker,
                                    aio_handler &&callback,
                                    int hash = 0,
                                    int64_t *pending_size = nullptr) = 0;
 
-    // get learn state in memory, including pending and writing mutations
-    // return true if some data is filled into writer
-    // return false if no data is filled into writer
-    // thread safe
+    // Get learn state in memory, including pending and writing mutations:
+    // - return true if some data is filled into writer
+    // - return false if no data is filled into writer
+    //
+    // Thread safe
     virtual bool get_learn_state_in_memory(decree start_decree, binary_writer &writer) const
     {
         return false;
     }
 
-    // only for private log
-    // get in-memory mutations, including pending and writing mutations
+    // Only for private log.
+    // get in-memory mutations, including pending and writing mutations.
     virtual void get_in_memory_mutations(decree start_decree,
                                          ballot current_ballot,
                                          /*out*/ std::vector<mutation_ptr> &mutations_list) const
     {
     }
 
-    // flush the pending buffer until all data is on disk
-    // thread safe
+    // Flush the pending buffer until all data is on disk.
+    //
+    // Thread safe.
     virtual void flush() = 0;
 
-    // flush the pending buffer at most once
-    // thread safe
+    // Flush the pending buffer at most once.
+    //
+    // Thread safe.
     virtual void flush_once() = 0;
 
 public:
     //
-    // ctors
+    // Ctors
     // when is_private = true, should specify "private_gpid"
```

Comment:
Can we remove is_private now, it must be true after the slog has been removed ?

Reply:
Yes, actually `slog` has been removed gradually, I'll create another PRs to drop `_is_private`.